ci-operator/step-registry/upi/install/azurestack/upi-install-azurestack-commands.sh

```diff
@@ -0,0 +1,284 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloud-credential-operator-config
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    release.openshift.io/create-only: "true"
+data:
+  disabled: "true"
+EOF
+
+# typical upi instruction
+python3 -c '
+import yaml;
+path = "manifests/cluster-scheduler-02-config.yml";
+data = yaml.full_load(open(path));
+data["spec"]["mastersSchedulable"] = False;
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+# typical upi instruction
+python3 -c '
+import yaml;
+path = "manifests/cluster-dns-02-config.yml";
+data = yaml.full_load(open(path));
+del data["spec"]["publicZone"];
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+INFRA_ID=$(python3 -c 'import yaml;data = yaml.full_load(open("manifests/cluster-infrastructure-02-config.yml"));print(data["status"]["infrastructureName"])')
+echo "${RESOURCE_GROUP}" > "${SHARED_DIR}/RESOURCE_GROUP_NAME"
+
+openshift-install create ignition-configs &
+
+set +e
+wait "$!"
+ret="$?"
+set -e
+
+cp "${dir}/.openshift_install.log" "${ARTIFACT_DIR}/.openshift_install.log"
+
+if [ $ret -ne 0 ]; then
+  exit "$ret"
+fi
+
+tar -czf "${SHARED_DIR}/.openshift_install_state.json.tgz" ".openshift_install_state.json"
+
+export SSH_PRIV_KEY_PATH="${CLUSTER_PROFILE_DIR}/ssh-privatekey"
+export OPENSHIFT_INSTALL_INVOKER="openshift-internal-ci/${JOB_NAME_SAFE}/${BUILD_ID}"
+
+date "+%F %X" > "${SHARED_DIR}/CLUSTER_INSTALL_START_TIME"
+RESOURCE_GROUP=$(cat "${SHARED_DIR}/RESOURCE_GROUP_NAME")
+
+az group create --name "$RESOURCE_GROUP" --location "$AZURE_REGION"
+
+az storage account create -g "$RESOURCE_GROUP" --location "$AZURE_REGION" --name "${INFRA_ID}sa" --kind Storage --sku Standard_LRS
+ACCOUNT_KEY=$(az storage account keys list -g "$RESOURCE_GROUP" --account-name "${INFRA_ID}sa" --query "[0].value" -o tsv)
+
+az storage container create --name files --account-name "${INFRA_ID}sa" --public-access blob --account-key "$ACCOUNT_KEY"
+az storage blob upload --account-name "${INFRA_ID}sa" --account-key "$ACCOUNT_KEY" -c "files" -f "bootstrap.ign" -n "bootstrap.ign"
+
+AZURESTACK_UPI_LOCATION="/var/lib/openshift-install/upi/azurestack"
+az deployment group create -g "$RESOURCE_GROUP" \
+  --template-file "${AZURESTACK_UPI_LOCATION}/01_vnet.json" \
+  --parameters baseName="$INFRA_ID"
+
+
+VHD_BLOB_URL="https://rhcossa.blob.ppe3.stackpoc.com/vhd/rhcos-49-84-202108221651.vhd"
+az deployment group create -g "$RESOURCE_GROUP" \
+  --template-file "${AZURESTACK_UPI_LOCATION}/02_storage.json" \
+  --parameters vhdBlobURL="$VHD_BLOB_URL" \
+  --parameters baseName="$INFRA_ID"
+
+az deployment group create -g "$RESOURCE_GROUP" \
+  --template-file "${AZURESTACK_UPI_LOCATION}/03_infra.json" \
+  --parameters baseName="$INFRA_ID"
+
+set +euE
+
+az network dns zone create -g "$RESOURCE_GROUP" -n "${CLUSTER_NAME}.${BASE_DOMAIN}"
+az network dns zone create -g "$RESOURCE_GROUP" -n "*.apps.${CLUSTER_NAME}"
```

Comment:
I don't think we need this zone. The *.apps record should be created in the zone created in the previous line